### ISSUE TITLE
Suggestion - Filling Up Jerry Cans

### ISSUE THREAD

**Issue by enutpeanut** [InZidiuZ/op-framework-issue-tracker#2615]
### Assurance

I have read the `README.md`

### Summary

Add a feature where we can pour petrol from one jerry can to another. I'm not entirely sure how you'd be able to do this but it's just an idea.

Another feature is to be able to fill up a jerry can at a petrol station, in the same way you fill up a car (e.g. hold out the jerry can next to the gas pump and fill it up). 

### Reason

Having eight jerry cans all at 20% fuel is annoying, and it would be nice to be able to merge them into just a couple. 
By being able to refill them, it rewards you to not throw them away, as it would be a bit cheaper than buying a new one.
I think it would be a nice addition and make the city more interesting.

**Comment by coalaura** [comment]
> I'm not entirely sure how you'd be able to do this

Me neither tbh. Don't really see how you could do it nicely

**Comment by enutpeanut** [comment]
> > I'm not entirely sure how you'd be able to do this
> 
> Me neither tbh. Don't really see how you could do it nicely

Yeah, I like the idea (it would also mean there aren't hundreds of jerry cans spread around the city between each tsunami) but I can't think of a good way to implement it. I'll give it some more thinking and get back to you :)

**Comment by enutpeanut** [comment]
@coalaura Thoughts on being able to fill them up at petrol stations? This would combat the same issue.

**Comment by vol5** [comment]
- Make them sellable at the bazaar: $25

- Add a breakdown spot for them 1 Jerry = 1 steel

**Comment by coalaura** [comment]
> @coalaura Thoughts on being able to fill them up at petrol stations? This would combat the same issue.

yeah this 4 sure but i meant the other thing

**Comment by coalaura** [comment]
> Make them sellable at the bazaar: $25

Also a nice idea

**Comment by roraraptor** [comment]
I would love to be able to refill them from gas pumps. 👍 

IMO it would also make more sense to

- Sell empty jerry cans inside the gas stations

**Comment by InZidiuZ** [comment]
This issue tracker is being shut down and replaced by Discord threads. If this issue is still relevant, create a thread on [Discord](https://discord.gg/opfw).
